modules/creating-recurring-etcd-backups.adoc

```diff
@@ -0,0 +1,258 @@
+[id="creating-recurring-etcd-backups_{context}"]
+== Creating recurring etcd backups
+
+Follow these steps to create automated recurring backups of etcd.
+
+Use dynamically-provisioned storage to keep the created etcd backup data in a safe, external location if possible. If dynamically-provisioned storage is not available, consider storing the backup data on an NFS share to make backup recovery more accessible.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. If dynamically-provisioned storage is available, complete the following steps to create automated recurring backups:
+
+.. Create a persistent volume claim (PVC) named `etcd-backup-pvc.yaml` with contents such as the following example:
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: etcd-backup-pvc
+  namespace: openshift-etcd
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 200Gi <1>
+  volumeMode: Filesystem
+  storageClassName: etcd-backup-local-storage
+----
+<1> The amount of storage available to the PVC. Adjust this value for your requirements.
++
+[NOTE]
+====
+Each of the following providers require changes to the `accessModes` and `storageClassName` keys:
+
+[cols="1,1,1"]
+|===
+|Provider|`accessModes` value|`storageClassName` value
+
+|AWS with the `versioned-installer-efc_operator-ci` profile
+|`- ReadWriteMany`
+|`efs-sc`
+
+|Google Cloud Platform
+|`- ReadWriteMany`
+|`filestore-csi`
+
+|Microsoft Azure
+|`- ReadWriteMany`
+|`azurefile-csi`
+|===
+====
```

Comment:
Does etcd backups prefer RWX volumes? If so - why?

Reply:
@Elbehery Can you answer @gnufied's question here?

Reply:
Hi @gnufied ,
Its not necessary to have "ReadWriteMany" . The only reason for RWX is to make sure that creation of testpod to access the location and verify backup file creation doesn’t cause “Multi attach” issue for the subsequent backup jobs.

For example, admin triggers scheduled/recurring jobs very frequently (every 1 minute). So, the backup will continue to get saved every minute. At the time, if the admin wants to access the external volume to check the backup file, then the subsequent backups fail with "multi attach" error. 
So, its for testing very frequent recurring backups that we have used RWX, otherwise RWO absolutely works fine as recurring backups doesn't create any parallel backup jobs.

Reply:
Thanks @sandeepknd  .. Gr8 answer 

@gnufied I also added in the PV manifest both modes `RWO` `RWX` and the claim should specify the accessMode

However, since this is `localVolume` it is attached only to one node